tests/test_Galactic.py::TestGalactic::test_proper_motion_identity failing

This test evaluates a position at the POSEPOCH and compares against the baseline position.  The threshold is set to 1e-11 arcsec.  Normally this passes but I've been having intermittent failures.  I don't know why it's intermittent (it should be deterministic) and only on some architectures.  I also don't know if this is due to #1748. 

Any ideas @abhisrkckl ?

Errors look like:
```
=================================== FAILURES ===================================
___________________ TestGalactic.test_proper_motion_identity ___________________

self = <test_Galactic.TestGalactic object at 0x7f0ba784c380>

    def test_proper_motion_identity(self):
        # sanity check that evaluation at POSEPOCH returns something very close to 0
        J0613_icrs = self.modelJ0613.coords_as_ICRS()
        J0613_icrs_alt = self.modelJ0613.coords_as_ICRS(
            epoch=self.modelJ0613.POSEPOCH.quantity.mjd
        )
        sep = J0613_icrs_alt.separation(J0613_icrs)
>       assert sep < 1e-11 * u.arcsec
E       assert <Angle 1.33203881e-14 deg> < (1e-11 * Unit("arcsec"))
E        +  where Unit("arcsec") = u.arcsec
```

--- a/CHANGELOG-unreleased.md
+++ b/CHANGELOG-unreleased.md
@@ -20,4 +20,5 @@ the released changes.
 - test for `pint.utils.split_swx()` 
 ### Fixed
 - `pint.utils.split_swx()` to use updated `SolarWindDispersionX()` parameter naming convention 
+- Fix #1759 by changing order of comparison
 ### Removed

--- a/tests/test_Galactic.py
+++ b/tests/test_Galactic.py
@@ -79,7 +79,7 @@ class TestGalactic:
         J0613_icrs_alt = self.modelJ0613.coords_as_ICRS(
             epoch=self.modelJ0613.POSEPOCH.quantity.mjd
         )
-        sep = J0613_icrs_alt.separation(J0613_icrs)
+        sep = J0613_icrs.separation(J0613_icrs_alt)
         assert sep < 1e-11 * u.arcsec
 
     def test_equatorial_to_galactic(self):

